Documentation/community/roadmap.rst

```diff
@@ -146,14 +148,16 @@ CI Test Improvements
 We have a comprehensive set of tests running in CI, but several contributors are
 currently working on `CI improvements`_ to make these more reliable and easier to
 maintain. This is a good area to get involved if you are interested in learning
-more about Cilium internals and development. 
+more about Cilium internals and development.
+
+Tetragon Security
+~~~~~~~~~~~~~~~~~
+
+Tetragon provides security observability through the JSON events and the Tetragon
+CLI for things like process execution, file access, network observability, and
+privileged execution.
 
-Documentation Improvements
-~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We would like to make this Cilium documentation :ref:`easier to
-navigate<season_of_docs>`, with more example configurations and links to
```

Comment:
Do we want to keep the GSoD dedicated page at all?

Reply:
for now, yes https://github.com/cilium/cilium/pull/21338#discussion_r974247249